.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.{scala,sbt}]
+indent_style = space
+indent_size = 2
```

Comment:
I generally add this to most projects as it makes editors that understand it behave themselves w.r.t. indentation - hope you don't mind

Reply:
Whatever it takes to keep the OSS contributions coming 😝